tests/impl.cpp

```diff
@@ -8020,7 +8020,85 @@ result_t test_mm_round_sd(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_round_ss(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    const float *_a = impl.mTestFloatPointer1;
+    const float *_b = impl.mTestFloatPointer1;
+    float f[4];
+    __m128 ret;
+
+    __m128 a = do_mm_load_ps(_a);
+    __m128 b = do_mm_load_ps(_b);
+    switch (i & 0x7) {
+    case 0:
+        f[0] = bankersRounding(_b[0]);
+        f[1] = _a[1];
+        f[2] = _a[2];
+        f[3] = _a[3];
```

Comment:
Maybe we can move the code out of the switch case since they are the same in different scenario?